src/PermissionsContext.tsx

```diff
@@ -13,15 +13,32 @@ import {
 } from "react-native-permissions"
 
 import { PermissionStatus, statusToEnum } from "./permissionStatus"
-
-export type ENAuthorization = `UNAUTHORIZED` | `AUTHORIZED`
-export type ENEnablement = `DISABLED` | `ENABLED`
 import gaenStrategy from "./gaen"
 
-export type ENPermissionStatus = [ENAuthorization, ENEnablement]
+type ENAuthorizationStatus = `UNAUTHORIZED` | `AUTHORIZED`
+type ENEnablementStatus = `DISABLED` | `ENABLED`
+export type ENPermissionStatus = [ENAuthorizationStatus, ENEnablementStatus]
+const initialENPermissionStatus: ENPermissionStatus = [
+  "UNAUTHORIZED",
+  "DISABLED",
+]
+
+const toAuthorizationEnablementStatus = (
+  enPermissionStatus: ENPermissionStatus,
+): ENAuthorizationEnablementStatus => {
+  const isENAuthorized = enPermissionStatus[0] === "AUTHORIZED"
+  const isENEnabled = enPermissionStatus[1] === "ENABLED"
+
+  return { authorization: isENAuthorized, enablement: isENEnabled }
```

Comment:
what do you think about
```suggestion
  return { authorized: isENAuthorized, enabled: isENEnabled }
```
?

Reply:
Yeah this is better! Naming is hard.